packages/babel-parser/test/fixtures/es2015/arrow-functions/.inner-parens-array-pattern-2/input.js

```diff
@@ -1 +0,0 @@
-([ [(a)] = [] ] = []) => {}
```

Comment:
I have enabled these tests on 00a776c34df0248bf4858fd47faa6c9772ae02c9 but forgot to remove them: https://github.com/babel/babel/pull/12327/files#diff-cb581cc9c63ffb0abb517f0508ad77cfd53fa50f53f16ee6e61eda11a0be0e3d